proposals/2022-04-min-commission-rate/README.md

```diff
@@ -0,0 +1,47 @@
+# Signaling Proposal: Upgrade the chain for introducing a min-commission-rate parameter and setting it to 3%
+
+
+## Proposal
+
+Previously, the proposal #6 "Increase minimum commission rate to 5%" was submitted but was rejected.<br>
+https://www.mintscan.io/medibloc/proposals/6
+
+This topic has been discussed for a very long time in the Cosmos ecosystem.<br>
+https://forum.cosmos.network/t/proposal-are-validators-charging-0-commission-harmful-to-the-success-of-the-cosmos-hub/2505
+
+Although proposal #6 was reasonable enough, MediBloc team also rejected it because we didn't have clear strategies and implementations for enforcing validators to increase their commission rate.
```

Comment:
```suggestion
Although proposal #6 was reasonable enough, MediBloc team rejected the proposoal because the team has yet to develop clear strategies to enforce the validators with low commision rate to increase their commission rate.
```